.travis.yml

```diff
@@ -46,21 +46,19 @@ matrix:
         packages:
           - google-chrome-stable
       cache:
-        yarn: true
         directories:
-          - $HOME/.yarn-cache
           - node_modules
       before_install:
         - sh -e /etc/init.d/xvfb start
         - export CHROME_BIN=/usr/bin/google-chrome
         - export DISPLAY=:99.0
-        - curl -o- -L https://yarnpkg.com/install.sh | bash
-        - export PATH="$HOME/.yarn/bin:$PATH"
-        - yarn global add @angular/cli
+        - npm install -g @angular/cli
         - ng --version
         - cd client
       install:
-        - yarn install
+        - npm install
       script:
-        - yarn run lint
-        - yarn run test --watch=false
+        - npm lint
+        - npm run test --watch=false
```

Comment:
`ng test --watch=false`
if you prefer to be consitent with using the ng command